.env.example

```diff
@@ -4,3 +4,4 @@ POSTGRES_DB = ""
 SECRET_KEY = ""
 ACCESS_TOKEN_EXPIRE_MINUTES = ""
 TOKEN_SIGNING_ALGORITHM = ""
+
```

Comment:
We dont need this line, if you will keep it it will cause merge conflict, so remove that line